source/extensions/compression/zstd/decompressor/zstd_decompressor_impl.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include "envoy/compression/decompressor/decompressor.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
+
+#include "source/extensions/compression/zstd/common/base.h"
+#include "source/extensions/compression/zstd/common/dictionary_manager.h"
+
+#include "zstd_errors.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Compression {
+namespace Zstd {
+namespace Decompressor {
+
+using ZstdDDictManager =
+    Common::DictionaryManager<ZSTD_DDict, ZSTD_freeDDict, ZSTD_getDictID_fromDDict>;
+using ZstdDDictManagerPtr = std::unique_ptr<ZstdDDictManager>;
+
+/**
+ * All zstd decompressor stats. @see stats_macros.h
+ */
+#define ALL_BROTLI_DECOMPRESSOR_STATS(COUNTER)                                                     \
```

Comment:
Why there is a `ALL_BROTLI_DECOMPRESSOR_STATS` in the zstd extension?

Reply:
Thanks, these code are copy from brotli module, and I have forgot to update it.